templates/container.cfg.xml.erb

```diff
@@ -12,6 +12,9 @@
 <%- if @via -%>
         via="<%= @via %>"
 <%- end -%>
+<%- if @via_header -%>
```

Comment:
This isn't going to work. This is trying to add the new option like it's an attribute. The new option is actually an entirely separate element within the xml config.

Reply:
Addressed this in amended commit